rest.http

```diff
@@ -31,8 +46,9 @@ Content-Type: application/json
 }
 
 # Save bookmark (no timestamp)
-# This will use current time
+# This will use current UTC time
```

Comment:
@rasendubi I didn't get this comment.

We use this function if there is no timestamp: [datetime.utcnow](https://docs.python.org/2/library/datetime.html#datetime.datetime.utcnow)

> Return the current UTC date and time